modules/nw-ingress-setting-a-custom-default-certificate.adoc

```diff
@@ -25,6 +23,13 @@ NAME      AGE
 default   10m
 ----
 
+[WARNING]
```

Comment:
Is this warning an elaboration of the first prereq? If so, I'd make it active and attach it to that prereq.

(Will the user know how to make sure the CA bundle is provided by the container userspace? (Also, userspace is only used once in the collection. Is there another way to convey this idea?)